### DISCUSSION TITLE
32 bit AIO (i386) image?

### DISCUSSION THREAD

**Discussion by Beryesa** [nextcloud/all-in-one#551] [Questions] [answered]
There's a i386 version for the nextcloud docker image, will AIO provide an i386 image? It would be really useful since setup is extremely simplified this way. I know 32 bit is not relevant for today but they have really low power consumption which is important for these kind of stuff. If not, will have to find my way through manually :/

**Comment by szaimen** [comment, marked as answer]
Hello, I am sorry to say but we will not build containers for any 32bit platform because of some incompatibilities of Nextcloud with 32bit.

> I know 32 bit is not relevant for today but they have really low power consumption which is important for these kind of stuff.

There are many devices out there that have a modern 64bit processor with a very low power comsumption which are also affordable. I'd recommend you to switch to one of these devices if you want to run AIO. Recommended from my side are e.g. Intel NUCs or the Raspberry Pi 4 with 4GB RAM or more and a SSD as main storage.

**Reply by Beryesa** [reply]
Thanks for the recommendations, I'm just I'm just re-considering the available hardware (and can't afford a new one to dedicate now). About the incompatibilities, can I still set it up manually or would that be just waste of time? Just looking for "Nextcloud files" functionality for the least :P 
Thanks for answering!

**Reply by szaimen** [reply]
Of course you can set up Nextcloud manually but I fear you will not be able to use this project for that...

**Reply by Beryesa** [reply]
Nextcloud still has the best "ecosystem" compared to other foss options, will still try my chance with the single docker image, have a great day/night :D

**Reply by szaimen** [reply]
yes, wish you luck with your setup! :)
